temporalio/client.py

```diff
@@ -6250,6 +6258,9 @@ async def heartbeat_async_activity(
             )
             if resp_by_id.cancel_requested:
                 raise AsyncActivityCancelledError()
+            if resp_by_id.activity_paused:
```

Comment:
Hrmm, looks like we never added "raises" docs to `AsyncActivityHandle.heartbeat`, wonder if we should now